### PR TITLE
Filter search results by judge(s)

### PR DESCRIPTION
**Pull request by nickmwangemi**
This PR aims to add functionality for filtering search results by judge(s).

Closes https://github.com/laws-africa/peachjam/issues/748

Demo:
https://www.loom.com/share/e5fdc8396c014424983f064a695e532a

### CODE REVIEW

File: peachjam/js/components/FindDocuments/index.vue [hunk 1]
```diff
@@ -209,7 +209,8 @@
       {{ $t('Matter type') }}
       {{ $t('Document nature') }}
       {{ $t('Language') }}
-      {{ $t('Year') }}
+      {{ $t('Judges') }}
```
Comment (actlikewill) [inline peachjam/js/components/FindDocuments/index.vue:212]: @nickmwangemi are we supposed to remove year here?

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Looks great.

I think it would be better if the judges facet comes just below the Court facet. The Court and Judge are closely logically linked.

**Comment by nickmwangemi** [comment]
> Looks great.
> 
> I think it would be better if the judges facet comes just below the Court facet. The Court and Judge are closely logically linked.

I've made the change:
image

**Comment by longhotsummer** [comment]
Looks good. I'll wait for #805 to be merged before approving, then we can deploy both.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**